ludwig/model_export/base_model_exporter.py

```diff
@@ -0,0 +1,22 @@
+from abc import ABC, abstractmethod
+
+import torch
+
+from ludwig.api import LudwigModel
+
+
+class BaseModelExporter(ABC):
+    def __init__(self, model):
+        self.model = model
+
+    @abstractmethod
+    def forward(self, x):
+        pass
+
+    @abstractmethod
+    def export_classifier_onnx(model_path, export_path):
+        pass
+
+    @abstractmethod
+    def quantize(path_fp32, path_int8):
```

Comment:
I think it's better design to call this:
```
def quantize(model_path, quantized_path):
```

The reason is that not all quantization is int8